controllers/add_secret.go

```diff
@@ -1,6 +0,0 @@
-package controllers
```

Comment:
These deletions can go in the previous commit for moving controllers to a single package

Reply:
That cant be done as deleting them requires the controller-runtime bump